code/modules/status_bar/status_bar.dm

```diff
@@ -17,4 +17,4 @@
 		return
 	client.status_bar_prev_text = text
 	winset(client, "mapwindow.status_bar",
```

Comment:
Btw, this can be refactored into the new `onstatus` feature in beta byond, which is completely client-side and low latency.

Reply:
use of 514 client features is blocked until #59940 is resolved. lummox is working on it now and has it narrowed down to the change that caused it, so hopefully this doesn't take too long, but it might involve him having to convert something to be threaded on the client side, so it could take too long.